docusaurus.config.js

```diff
@@ -83,6 +83,17 @@ const config = {
         sidebarPath: './sidebars.js',
       },
     ],
+    [
+      '@docusaurus/plugin-content-docs',
+      {
+        id: 'ed-fi-badges',
+        path: 'docs/ed-fi-badges',
+        editUrl: ({ docPath }) =>
+          `https://github.com/ed-fi-alliance-oss/ed-fi-alliance-oss.github.io/tree/main/${docPath}/`,
+        routeBasePath: 'docs/ed-fi-badges',
+        sidebarPath: './sidebars.js',
```

Comment:
Previously, I had the Badging section laid out under "partners" above. Is there an advantage to using `docs/ed-fi-badges` instead? Or perhaps simplified to `docs/badging`, or `docs/partners/badging` if the "docs" part is really important _and_  if there is value to keeping `partners` in the URL path.